src/components/LetterCasePicker.tsx

```diff
@@ -43,35 +43,34 @@ const LetterCasePicker: FC<{ fontSize: number; style?: React.CSSProperties }> =
     <div className={css({ userSelect: 'none' })}>
       <div
         ref={ref}
-        style={style}
-        className={css({
-          background: { base: '#ebebeb', _dark: '#141414' },
-          borderRadius: '3',
-          display: 'inline-block',
-          padding: '0.2em 0.25em 0.25em',
-          position: 'relative',
-          ...(overflow.left ? { left: `${overflow.left}` } : { right: `${overflow.right}` }),
-        })}
+        style={{ ...(overflow.left ? { left: `${overflow.left}` } : { right: `${overflow.right}` }) }}
+        className={css(
+          {
+            background: { base: '#ebebeb', _dark: '#141414' },
+            borderRadius: '3',
+            display: 'inline-block',
+            padding: '0.2em 0.25em 0.25em',
+            position: 'relative',
+          },
+          cssRaw,
+        )}
       >
         <TriangleDown
-          fill={colors.fgOverlay90}
+          fill={token('colors.fgOverlay90')}
           size={fontSize}
           cssRaw={{ position: 'absolute', width: '100%' }}
-          style={{
-            ...(overflow.left ? { left: -overflow.left } : { right: -overflow.right }),
-            top: -fontSize / 2,
-          }}
+          style={{ ...(overflow.left ? { left: -overflow.left } : { right: -overflow.right }), top: -fontSize / 2 }}
         />
 
         <div aria-label='letter case swatches' className={css({ whiteSpace: 'wrap' })}>
           {casingTypes.map(type => (
             <div
               key={type}
               title={type}
-              style={{ border: `solid 1px ${selected === type ? `${colors.fg}` : 'transparent'}` }}
               className={css({
                 margin: '2px',
                 lineHeight: '0',
+                border: selected === type ? `solid 1px {colors.fg}` : `solid 1px transparent`,
```

Comment:
Just a note: I checked if this border would cause the same issue (https://github.com/cybersemics/em/issues/2508) as with the ColorPicker, but it does not 👍

Reply:
This is the strangest bug I've seen in a while 😵‍💫